eu.jumplink.Easy6502.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>eu.jumplink.Easy6502</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>CC-BY-4.0</project_license>
+  <name>Easy6502</name>
+  <summary>Modern 6502 Assembly Learning Environment</summary>
+  <description>
+    <p>
+      Easy6502 is a modern, native Adwaita application that provides a complete learning environment for 6502 assembly language programming. Built with GJS and TypeScript, this application brings the classic Easy6502 tutorial by Nick Morgan to your desktop as a beautiful, integrated experience.
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>Interactive Tutorial: A comprehensive step-by-step guide to learning 6502 assembly language, with explanations of all core concepts</li>
+      <li>Code Editor: Write and edit your 6502 assembly code with syntax highlighting</li>
+      <li>Assembler &amp; Debugger: Assemble your code and debug it with a powerful built-in debugger showing registers, flags, and memory in real-time</li>
+      <li>Visual Game Console: See your code in action on a virtual display, perfect for creating simple games and graphics</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">eu.jumplink.Easy6502.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/JumpLink/Easy6502/refs/heads/main/misc/screenshots/1.png</image>
```

Comment:
screenshot URL should never reference the tip of branch as it is not immutable.

(applies to all)